frontend/src/app/services/event-list.ts

```diff
@@ -167,18 +167,18 @@ export const eventTriggers: EventTrigger[] = [
   {
     events: [
       {
-        title: 'První případ nákazy koronavirem!',
-        text: '<p>Nemocí Covid-19 se v Česku nakazil první člověk a řešení této situace máte teď ve svých rukou. Než se do toho pustíte, měli byste vědět tohle:</p>\
-<ul>\
+        title: 'První případ nákazy koronavirem v Česku!',
+        text: '<p>Řešení této situace máte teď ve svých rukou. Pamatujte, že:</p>\
+<ul style="padding-top: 0.5rem; padding-bottom: 0.5rem">\
```

Comment:
Do not do this, please, solve it systematically. The html should be style-less.

Reply:
Presunu do stylu.